### ISSUE TITLE
Be able to use percentages % of remaning space when setting flexibility

### ISSUE THREAD

**Issue by Jeremy-Knudsen** [WestonThayer/Bloks#15]
If I have two spacers within a BlokGroup, I'd like to be able to specifically set how much each spacer should flex (e.g. 33% and 67% remaining space).

**Comment by WestonThayer** [comment]
The flexibility property maps to CSS's [flex-grow](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow), so I think you can do this with a flexibility value of `1` for the first item and `2` for the second item.

Edit: or is your feedback more that the `1` and `2` values are confusing, and percentages would be more intuitive?

**Comment by Jeremy-Knudsen** [comment]
Yes, it's confusing but perhaps the reason why is because:

1) the help tooltip comes across as if there are only 3 options to choose from--1, 2, and unset--which made me think a radio button group would be a better UI for this if those were the only three options. But what I think you are saying is that this actually maps to the flex-grow parameter which can be any number really. Is that right?

2) I did not know that the flexibility parameter in Bloks maps one-to-one with CSS's "flex-grow" attribute. If so, then I would consider updating the help tip to show the flex-grow syntax for acceptable values per the flex-grow CSS documentation you provided.

**Comment by WestonThayer** [comment]
Correct, it maps to `flex-grow`, so you can use any positive integer. Thanks for the feedback. I keep wondering if there's some better way to represent flexibility...
